edge-util/src/Microsoft.Azure.Devices.Edge.Storage.RocksDb/RocksDbOptionsProvider.cs

```diff
@@ -45,6 +45,15 @@ public class RocksDbOptionsProvider : IRocksDbOptionsProvider
         // column families whose memtables are backed by the oldest live WAL file
         const ulong DefaultMaxTotalWalSize = 512 * 1024 * 1024;
 
+        // max_manifest_file_size = 1G (bytes)
+        //
+        // The MANIFEST-<sequence number> file holds the history of operations.
+        // When the file exceeds the configured max size, a new file is created,
+        // and the previous file is marked obsolete and is deleted in the next compaction.
+        //
+        // Upstream defaults to 1G.
+        const ulong DefaultMaxManifestFileSize = 1024 * 1024 * 1024;
```

Comment:
THis seems large.. should we cap it to say, 64Mb or so?

Reply:
Done.